CMakeLists.txt

```diff
@@ -818,6 +818,21 @@ elseif(IS_DIRECTORY .git AND GIT)
     )
 endif()
 
+set(TZCODE_SOURCE_DIR ${PROJECT_SOURCE_DIR}/third_party/tz)
+if (EXISTS "${TZCODE_SOURCE_DIR}/.git" AND GIT)
+    execute_process(COMMAND ${GIT_EXECUTABLE} describe --tags --dirty
+                    WORKING_DIRECTORY ${TZCODE_SOURCE_DIR}
+                    OUTPUT_VARIABLE TZDATA_VERSION
+                    RESULT_VARIABLE TZDATA_ERROR_CODE
+                    OUTPUT_STRIP_TRAILING_WHITESPACE)
+endif()
```

Comment:
I doubt if it works if we build tarantool from a source tarball.

I guess that tarball packing should save the `VERSION` file in the `third_party/tz` directory and the build process should catch it.

Reply:
Added a constant with version to the root CMakeLists.txt and added a test that compares `tarantool.build.tzdata_version` and hardcoded version. (as discussed verbally.)

Reply:
Thanks!